infra.bs

```diff
@@ -296,30 +296,30 @@ for working with them, in order to create common ground.
 <h3 id=lists>Lists</h3>
 
 <p>A <dfn export>list</dfn> is a specification type consisting of a finite ordered sequence of
-items.
+<dfn for="list,stack,queue,set" lt=item export>items</dfn>.
 
 <p>For notational convenience, a literal syntax can be used to express <a>lists</a>, by surrounding
-the list contents by « » characters and separating list items with a comma. An indexing syntax can
-be used by providing a zero-based index into a list inside square brackets.
+the list contents by « » characters and separating list <a for=list>items</a> with a comma. An
+indexing syntax can be used by providing a zero-based index into a list inside square brackets.
 
 <p class=example id=example-list-notation>Let |example| be the <a>list</a> « "<code>a</code>",
 "<code>b</code>", "<code>c</code>", "<code>a</code>" ». Then |example|[1] is the <a>string</a>
 "<code>b</code>".
 
 <hr>
 
-<p>To <dfn export for=list>append</dfn> to a <a>list</a> that is not an <a>ordered set</a> is to add
-the given item to the end of the list.
+<p>To <dfn export for=list>append</dfn> to a <a>list</a> that is not an <a>ordered set</a> is to
+add the given <a for=list>item</a> to the end of the list.
 
 <p>To <dfn export for=list>prepend</dfn> to a <a>list</a> that is not an <a>ordered set</a> is to
-add the given item to the beginning of the list.
+add the given <a for=list>item</a> to the beginning of the list.
 
 <p class=note>The above definitions are modified when the <a>list</a> is an <a>ordered set</a>; see
 below for <a for=set lt=append>ordered set append</a> and
 <a for=set lt=prepend>ordered set prepend</a>.
 
-<p>To <dfn export for=list,set>remove</dfn> an item from a <a>list</a> is to remove all items
-from the list that match a given condition, or do nothing if none do.
+<p>To <dfn export for=list,set>remove</dfn> an <a for=list>item</a> from a <a>list</a> is to remove
+all items from the list that match a given condition, or do nothing if none do.
```

Comment:
Oh, it seemed the standard was not to link to the same dfn twice in the same paragraph.